tools/rna_tools/targetfinder/TargetFinder.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="targetfinder" name="targetfinder" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.07">targetfinder</requirement>
+        <requirement type="package" version="35.4.12">fasta</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+
+    <command><![CDATA[
+        targetfinder.pl
+            -s "$s"
+            -d $d
+            -q "$q"
+            -c $c
+            -t $t
+            -p "$p"
+            $r
+            > $output1
+    ]]></command>
+    <inputs>
+        <param label="Small RNA sequence (RNA or DNA, 5'->3')" help="(-s)" name="s" type="text"/>
+        <param label="Target sequence database file (FASTA-format)" help="(-d)" name="d" type="data" format="fasta" />
+        <param label="Query sequence name (DEFAULT = 'query')" help="(-q)" name="q" type="text" value="query" />
+        <param label="Prediction score cutoff value (DEFAULT = 4)" help="(-c)" name="c" type="float" value="4.0" />
+        <param label="Threads for parallel Smith-Waterman searches (DEFAULT = 1)" help="(-t)" name="t" type="integer" value="1" />
+        <param label="Output format for small RNA-target pairs." help="(-p)" name="p" type="select" >
+                <option value="classic">classic</option>
+                <option value="gff">gff</option>
+                <option value="json">json</option>
+                <option value="table">table</option>
+        </param>
+        <param label="Search reverse strand for targets? Use this option if the database is genomic DNA." help="(-r)" name="r" type="boolean" falsevalue="" truevalue="-r" checked="false" />
+    </inputs>
+    <outputs>
+        <data label="Predicted targets by targetfinder" name="output1" format="data"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="s" value="UGCCAAAGGAGAUUUGCCCUG"/>
+            <param name="d" value="Antirrhinum_majus.mRNA.EST.fasta"/>
+            <param name="q" value="miR399a"/>
+            <param name="r" value="-r"/>
+            <param name="c" value="4.0"/>
+            <param name="t" value="1"/>
+            <param name="p" value="classic"/>
+            <output name="output1" file="miR399a_predicted_targets.txt"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+**What it does**
```

Comment:
The help should be really improved. More is better ... please note that this is how we can train our users to do the correct thing and help them. As better the help is the less questions they have.